packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/selectBox.tests.js

```diff
@@ -6481,6 +6481,29 @@ QUnit.module('valueChanged handler should receive correct event', {
     });
 });
 
+QUnit.module('onKeyDown', () => {
```

Comment:
We already have a "Keyboard navigation" module, let's just move this test there because I don't think we gonna have a lot of tests exactly for a keydown event

Reply:
[603415f](https://github.com/DevExpress/DevExtreme/pull/27660/commits/603415f3c5b1bed45ea621e774cd2f09942e9ac3)